easybuild/easyconfigs/c/CompareM/CompareM-0.0.23-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'CompareM'
+version = '0.0.23'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/dparks1134/CompareM'
+description = "A toolbox for comparative genomics."
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('matplotlib', '2.2.3', versionsuffix),
+]
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+use_pip = True
+
+exts_list = [
+    ('future', '0.18.2', {
+        'checksums': ['b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d'],
+    }),
+    ('biolib', '0.0.46', {
+        'checksums': ['232e0d02d4b0a759f94c613f22ab3a9e5bb9230666308446fa8e08c7597db156'],
+    }),
+    ('comparem', version, {
+        'checksums': ['1f4b693298d92499bf7c3174220e06260e3c446273517c43781b14b284155f5b'],
```

Comment:
The install instructions for comparem sys that it assumes
prodigal and diamond to be in the path.

They are missing here.

Reply:
fixed in 644260d